### ISSUE TITLE
Hit testing for plot titles

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#1420]
Implement hit testing for titles and sub-titles

**Comment by objorke** [comment]
I think this can be done by adding two rectangle based `PlotElement` s (stored and updated in the `PlotModel` class) to https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/PlotModel/PlotModel.cs#L1206

**Comment by silviubogan** [comment]
@objorke Should I do this in my first (currently only) PR? Thank you.

**Comment by objorke** [comment]
@silviubogan yes, but please make a separate PR with this change only, to make it easier to accept/merge. 
Can you implement this without a wrapper class. I think it can be implemented with new `TitleElement` and `SubtitleElement` classes which are stored internally on `PlotModel`. Hopefully with no need to make any breaking changes.
